src/lib.rs

```diff
@@ -627,7 +630,7 @@ impl Cache {
 
     fn scoped(mut self, scope: String) -> Self {
         assert!(self.scope.is_none());
-        self.scope = Some(scope);
+        self.scope = Some(general_purpose::STANDARD_NO_PAD.encode(scope));
```

Comment:
Let's do the encoding [where the path is constructed](https://github.com/dimo414/bkt/blob/b01ecb612b2aa37e84b79a0340d23c1d165bd655/src/lib.rs#L667), instead of here